src/kOS/Communication/ProcessorManager.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using kOS.Module;
+using kOS.Safe.Compilation;
+using kOS.Safe.Persistence;
+using kOS.Safe.Utilities;
+
+namespace kOS.Communication
+{
+    public class ProcessorManager
```

Comment:
This looks like a duplicate of `kOS.InterProcessor.ProcessorManager`, but I don't see the old manager deleted.  If we remove the run command, is this even needed?  It doesn't appear to connect to your messaging stuff directly.